### PR TITLE
`amount` option renamed to `deposit` (with alias)

### PR DESCRIPTION
**Pull request by volovyk-s**
I think that `deposit` is a better word for a deposit :)

Options list was out of date in README

off top:  probably we should delete dead EVM commands in the next release, so basically we can do that in the next PR

### CODE REVIEW

File: commands/call.js [hunk 1]
```diff
@@ -13,10 +13,11 @@ module.exports = {
             type: 'string',
             default: '100000000000000'
         })
-        .option('amount', {
-            desc: 'Number of tokens to attach (in NEAR)',
+        .option('deposit', {
+            desc: 'Number of tokens to attach (in NEAR) in a function call',
```
Comment (mehtaphysical) [inline commands/call.js:17]:
```suggestion
            desc: 'Number of tokens to attach (in NEAR) to a function call',
```

**Review by mehtaphysical — APPROVED** [review]
Looks good. Just one suggestion

File: commands/call.js (hunk 1 above)
Comment (volovyk-s) [reply commands/call.js]: Thanks!

**Review by thisisjoshford — APPROVED** [review]
LGTM. Let me know when you merge and I'll update docs :)

**Pull request merged**